cloudnetpy_qc/quality.py

```diff
@@ -171,6 +147,108 @@ def _read_config_keys(self, config_section: str) -> np.ndarray:
         keys = METADATA_CONFIG[config_section][field].split(",")
         return np.char.strip(keys)
 
+    def _get_required_variables(self) -> list[dict]:
+        return [
+            {name: var}
+            for name, var in variables.items()
+            if var.required is not None and self.cloudnet_file_type in var.required
+        ]
+
+    def _get_required_variable_names(self) -> set:
+        required_variables = self._get_required_variables()
+        return set().union(*(d.keys() for d in required_variables))
```

Comment:
Eikö tätä voisi yksinkertaistaa jotenkin näin?
```suggestion
    def _get_required_variables(self) -> dict:
        return {
            name: var
            for name, var in variables.items()
            if var.required is not None and self.cloudnet_file_type in var.required
        }

    def _get_required_variable_names(self) -> set:
        required_variables = self._get_required_variables()
        return set(required_variables.keys())
```

Reply:
:+1: